docs/piaf/reader/report_january.md

```diff
@@ -3,19 +3,26 @@
 ## Context
 At the dawn of launching a new project on the CRPA data to further test the opportunity of using PIAF in the context of the French administration, we were facing concerns regarding the standardization of the data we were to feed to PIAF.
 
-We want to be able to launch a new instance of PIAF with as few human intervention as possible. However the performance of the whole pipeline is much influenced by the size of the knowledge base available for searching. We also know that many parameters will influence the answer retrieval in terms of quality (number of correct retrievals) and response-time. These parameters can be : document length, number of documents retreived, number of answers requested per documents, among others.
+We want to be able to launch a new instance of PIAF with as few human interventions as possible.
```

Comment:
I don't think that we want bullet points here. I believe that these paragraphs are not an enumeration of events/ideas but a single logical sequence.

Reply:
I did not want to create bullet points and I think the output will be exactly the same. I understand the idea of logical sequence, it's just that having some newlines in the .md simplifies the document's readability/git diffs (IMHO).

I can revert that if you prefer having only one line in the markdown

Reply:
oh I see! I did not check the output. Having single lines seems better indeed :+1: